.github/workflows/build-scala2.11-spark2.4.yml

```diff
@@ -0,0 +1,24 @@
+name: Build Scala 2.11, Spark 2.4
```

Comment:
Maybe it would be easier to have just one file. Look at [Cobrix's build.yml](https://github.com/AbsaOSS/cobrix/blob/master/.github/workflows/build.yml)

Reply:
There's the added benefit of potentially adding two separate build badges on the readme. That's why I put it in two separate files

Reply:
In that case, beware of the long name being badly parsed. Again, Cobrix had this issue and switched to a single file.